sdk/core/Microsoft.Extensions.Azure/src/Microsoft.Extensions.Azure.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <Description>Azure Client SDK integration with Microsoft.Extensions libraries</Description>
     <AssemblyTitle>Azure Client SDK integration Microsoft.Extensions</AssemblyTitle>
-    <VersionPrefix>1.0.0</VersionPrefix>
+    <VersionPrefix>1.0.1</VersionPrefix>
```

Comment:
This one did not ship in master when we created this fork so we should undo this until those changes come into this repo.